### DISCUSSION TITLE
What method should i use to get the best performance for my game server? (cluster, child process or worker threads)

### DISCUSSION THREAD

**Discussion by TheMetaverseEngineer** [dimforge/rapier#458] [Q&A]
Hello,
I'm using rapier in my Node.js app. In my game, I put every three player at a socket.io room and they play together. What is the best method for increasing the performance of my app? Should I use worker threads for my game class and run every game in a thread, or should I use children processes? What about clustering?

**Comment by Glavin001** [comment]
@TheMetaverseEngineer what did you end up with for a solution?
